files/default/sql/ddl/3.4.0__initial_tables.sql

```diff
@@ -2031,7 +2031,7 @@ CREATE TABLE IF NOT EXISTS `hopsworks`.`training_dataset_filter_condition` (
     `filter_value_fg_id` INT(11) NULL,
     PRIMARY KEY (`id`),
     CONSTRAINT `tdfc_training_dataset_filter_fk` FOREIGN KEY (`training_dataset_filter_id`) REFERENCES `training_dataset_filter` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
-    CONSTRAINT `tdfc_feature_group_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION
+    CONSTRAINT `tdfc_feature_group_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE SET NULL ON UPDATE NO ACTION
```

Comment:
why "SET NULL" and not "CASCADE"?

Reply:
It is the same as https://github.com/logicalclocks/hopsworks-chef/blob/3a2a53be29e368ce97ea3cf4f28d11f4ea6f5d6b/files/default/sql/ddl/3.4.0__initial_tables.sql#L1333